src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,18 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: getMaxWidth($cardWidth, $cardSideGaps, $catalogSidePaddings);
+
+  &__product {
+    margin-right: $cardSideGaps;
+    margin-top: 46px;
```

Comment:
it is better to use margin bottom everywhere

Reply:
fixed